Restaurant/src/RandomRestaurantServlet.java

```diff
@@ -0,0 +1,45 @@
+import java.io.*;
+import java.util.List;
+import java.util.Random;
+import java.util.ArrayList;
+import javax.servlet.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.WebServlet;
+
+@WebServlet(loadOnStartup = 1, urlPatterns = { "/suggest/restaurants/random" })
+public class RandomRestaurantServlet  extends HttpServlet {
+	
+
+	public Restaurant getRandomRestaurant(List<Restaurant> list) {
+		return list.get(new Random().nextInt(list.size()));
+	}
+
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		PrintWriter out = response.getWriter();
+		List<Restaurant> entries = (List<Restaurant>) getServletContext().getAttribute("entries");
```

Comment:
should probably rename "entries" to be something more specific like "restaurants" so that you can rum multiple applications at once.